packages/core/framework/src/http/types.ts

```diff
@@ -123,12 +123,7 @@ export interface MedusaRequest<Body = unknown>
    */
   filterableFields: Record<string, unknown>
   includes?: Record<string, boolean>
-  /**
-   * An array of fields and relations that are allowed to be queried, this can be set by the
-   * consumer as part of a middleware and it will take precedence over the defaultAllowedFields
-   * @deprecated use `allowed` instead
-   */
-  allowedFields?: string[]
+
   /**
    * An array of fields and relations that are allowed to be queried, this can be set by the
    * consumer as part of a middleware and it will take precedence over the defaultAllowedFields set
```

Comment:
suggestion: we might need to update the description slightly, and maybe mention req.allowed explicitly wdyt?

Reply:
Not sure I get it, which comment should I update?

Reply:
this one where we mention among other things the `defaultAllowedFields` no?
```text
An array of fields and relations that are allowed to be queried, this can be set by the
   * consumer as part of a middleware and it will take precedence over the defaultAllowedFields set
   ```

Reply:
Done